aiida/orm/nodes/data/structure.py

```diff
@@ -18,6 +18,7 @@
 import itertools
 import copy
 from functools import reduce
+from packaging import version
```

Comment:
Thanks for the update @astamminger . One question: is there a reason you prefer the dependency `packaging` over the standard library `distutils`?

Reply:
Was about to write a comment on that but you were too fast :D Mainly the reason was that AFAIK ``StrictVersion`` provided by ``distutils`` does not comply to PEP440 so I tought``packaging`` might be a better alternative (But I'm happy to change that if you prefer to use ``distutils``)

Reply:
Thanks, I didn't know that. It seems indeed that `packaging` seems the more reliable option. I was just worried about it being third-party and therefore adding an implicit dependency, but since it gets installed by `setuptools` that is ok

Reply:
From looking at the list of maintainers on PyPI, ``packaging`` seems pretty solid. We should add it as an explicit dependency in ``setup.json`` if we use it explicitly, though.

Reply:
Could you add the dependency to the `atomic_tools` extra @astamminger ? Since it is only used in combination with `pymatgen` I think it is ok to just put it in that extra

Reply:
Shouldn't we move the import to the function itself then? Might be easier to just put it in the general dependencies.

Reply:
We could also use ``pkg_resources.parse_version()`` which is distributed with ``setuptools`` so we can avoid adding an additional dependency plus being PEP440 conform. What do you think?

Reply:
Go for it 👍 Let's try not to [bike-shed](https://en.wikipedia.org/wiki/Law_of_triviality) too hard here..

I have a feeling the coveralls can be a bit flaky sometimes.